ts/modules/export-data.src.ts

```diff
@@ -250,6 +251,18 @@ Highcharts.setOptions({
              */
             columnHeaderFormatter: null,
 
+            /**
+            * When several labels are assigned to a specific point,
+            * they will be displayed in one field in the table.
+            *
+            * @sample highcharts/export-data/concatenate-point-annotations/
+            *         Concatenate point annotations.
+            *
+            * @since   next
+            * @requires modules/annotations
+            */
+            concatenatePointAnnotations: false,
```

Comment:
IMHO the parameters is too long. 

Maybe linkedPointAnnotations or linkPointAnnotations is more intuitive / user-friendly ?

Reply:
I created it based on this discussion: https://github.com/highcharts/highcharts/issues/12789#issuecomment-585633261
Of course, we can change it. @KacperMadej any thoughts?

Reply:
How about `joinAnnotations: boolean|string` ? String can be the glue in the concatenated string (right now called `annotationSeparator = '; '`)

Reply:
or even `includeAnnotations` or `addAnnotations`?